src/build/__init__.py

```diff
@@ -21,9 +21,12 @@
 if sys.version_info < (3,):
     FileNotFoundError = IOError
     PermissionError = OSError
+else:
+    unicode = str
 
 
 ConfigSettings = Dict[str, Union[str, List[str]]]
+Path = Union[str, bytes, unicode]
```

Comment:
Why not use ``Union[bytes, Text]``?

Reply:
That doesn't accept normal strings on Python 2, does it?

Reply:
`Union[bytes, Text]` does.

Reply:
`Union[bytes, Text]` is synonymous with `Union[str, bytes, unicode]`.  According to the typeshed stub `subprocess.check_call` accepts both bytes and Unicode in Python 2: https://github.com/python/typeshed/blob/master/stdlib/2/subprocess.pyi#L8.

Reply:
> `Text` is an alias for `str`. It is provided to supply a forward compatible path for Python 2 code: in Python 2, `Text` is an alias for `unicode`.

That's not how I interpret the documentation.

Reply:
Exactly, so ``Union[bytes, Text]`` is essentially ``Union[str, bytes, unicode]``; what you have. On Python 3 IMHO we should only accept ``Text`` (which resolves to just ``str``).

Reply:
`subprocess` accepts `bytes` in Python 3 too so it shouldn't matter unless you have a preference for Unicode.

Reply:
```
Python 2.7.18 (default, Sep  5 2020, 11:17:26)
[GCC 10.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import typing
>>> typing.Text
<type 'unicode'>
```

Am I missing something? Using `typing.Text` would not include `str` on Python 2, right?

Reply:
No, but `str is bytes` and the proposed format is `Union[bytes, Text]`.  In Python 3 this resolves into `Union[bytes, str]`; in Python 2 it resolves into `Union[bytes, unicode]`.

Reply:
Ah, nevermind. I am dumb :sweat_smile: